src/main/java/com/wanderersoftherift/wotr/init/WotrItems.java

```diff
@@ -107,6 +109,65 @@ public class WotrItems {
     public static final DeferredItem<Item> COLOR_HELMET = ITEMS.register("color_helmet",
             () -> new WotrArmor(EquipmentSlot.HEAD, "color_helmet", 165));
 
+    // Essence
+    public static final DeferredItem<Item> EARTH_ESSENCE = registerEssence(
+            EssenceType.EARTH.getName().toLowerCase() + "_essence");
```

Comment:
Please use `toLowerCase(Locale.ROOT)` - otherwise will break for people running with some languages (example: capital `I` lowercases to `ı` in Turkish). Could move the EssenceType -> id logic into the registerEssence method and just pass the enum.